test/cli.js

```diff
@@ -325,7 +325,9 @@ test('watcher does not rerun test files when they write snapshot files', t => {
 				killed = true;
 			}, 500);
 		} else if (passedFirst && !killed) {
-			t.is(buffer.replace(/\s/g, ''), '');
+			const rerunMessage = `To rerun all tests, type 'r', followed by Enter\nTo update snapshots used in the previous tests, type 'u', followed by Enter\n`;
+			const rerunMessageWithoutWhitespace = rerunMessage.replace(/\s/g, '');
+			t.is(buffer.replace(/\s/g, ''), rerunMessageWithoutWhitespace);
```

Comment:
@novemberborn I'm not sure if what I did (asserting this) is the right way to do this

Reply:
Most tests use [`t.match`](http://www.node-tap.org/asserts/#tmatchfound-pattern-message-extra) for this. You could do an assertion for each line.